src/appConfigs/schema.ts

```diff
@@ -14,6 +17,20 @@ export const typeDefs = gql`
     toUid: ID!
   }
 
+  type ContactWith {
+    uid: ID!
+    met_on: String!
```

Comment:
@kavyvetri Can this be type: `DateTime!`?

Reply:
because we return comma separated dates, I changed to string. Let me know if you think array of dates will be helpful. we can do that.